src/table/body-cell/styles.scss

```diff
@@ -147,28 +148,21 @@ $cell-negative-space-vertical: 2px;
   }
   &:first-child {
     border-inline-start: $border-placeholder;
+    @include cell-padding-inline-start($cell-edge-horizontal-padding);
```

Comment:
Moved from `first-child:not(.is-visual-refresh)`. There is no reason to include the ":not(.is-visual-refresh)" because first column padding for VR is overridden anyways.